### PR TITLE
Add Scope for Ordering

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Add scope to enabling find the most recent phq assessment

[Finished: #100127766]

### CODE REVIEW

File: app/models/phq_assessment.rb [hunk 1]
```diff
@@ -11,6 +11,12 @@ class PhqAssessment < ActiveRecord::Base
   validates :release_date, uniqueness: { scope: :participant_id }
   validate :scores_valid
 
+  scope :when_updated, -> { order(updated_at: :desc) }
```
Comment (ericcf) [inline app/models/phq_assessment.rb]: Would you consider renaming to something more intuitive such as `latest_updated`?

Comment (michaelwehrley) [inline app/models/phq_assessment.rb]: that is perfecto.  :bow: thanks!

**Comment by michaelwehrley** [comment]
@ericcf all set.

**Comment by ericcf** [comment]
:cake:

**Pull request merged**